data/Akshaykk12.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "Akshay Kudalkar",
+  "type": "personal",
+  "bio": "Full stack Designer || Blockchain Enthusiast || Currently learning AI || Always Learning || Open Source ",
+  "links": [
+    {
+      "name": "Twitter",
+      "url": "https://twitter.com/Akshay12kk",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Casual Instagram",
+      "url": "https://www.instagram.com/akshaykk_12/",
+      "icon": "FaInstagram"
+    },
+    {
+      "name": "LinkedIn",
+      "url": "https://www.linkedin.com/in/akshay-kudalkar/",
+      "icon": "FaLinkedin"
+    },
+    {
+      "name": "GitHub",
+      "url": "https://github.com/Akshaykk12",
+      "icon": "FaGit"
```

Comment:
Just change the icon from `FaGit` to `FaGithub`
``` suggestion
      "icon": "FaGithub"
```

Reply:
Thank you